.github/workflows/pythonpackage.yml

```diff
@@ -28,4 +28,4 @@ jobs:
         pip install -r requirements/requirements-test.txt
     - name: Test with pytest
       run: |
-        pytest -m 'not (gpu or serial)' --cov src/gluonts --cov-report=term --cov-report xml test
+        pytest --timeout=60 -m 'not (gpu or serial)' --cov src/gluonts --cov-report=term --cov-report xml test
```

Comment:
```suggestion
        pytest -m 'not (gpu or serial)' --cov src/gluonts --cov-report=term --cov-report xml test
```

Reply:
Let's just not add these I guess, I think this will prevent the windows builds from failing